core/main_test.ts

```diff
@@ -3168,6 +3244,110 @@ actions:
         ])
       );
     });
+
+    suite("incremental table on schema change", () => {
```

Comment:
Similarly with this - also the suite name is identical, even though this one is for action configs files rather than SQLX config blocks

Reply:
Similarly, fixed this.